src/index.js

```diff
@@ -1,12 +1,7 @@
-import _ from 'lodash';
 import './style.css';
 
 function component() {
   const element = document.createElement('div');
-
```

Comment:
- [ ] I would recommend using arrow functions instead of the function keyword. Please fix it.

Reply:
@Dipeshtwis please explain how is this a requirement ?

Reply:
Since it's not required function, I have removed it

Reply:
Hi @benjp009,

Sorry, I have not explained earlier but one of the basic requirements for this project is to learn and implement ES6 syntax where we have to learn how to use arrow function instead of function keyword.